src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -187,14 +187,23 @@ export const validateSubmission = async (
     .map(() => next())
     .mapErr((error) => {
       logger.error({
-        message: 'Error processing responses',
+        message:
+          'Error processing responses, but proceeding with submission as submission have been validated client-side',
```

Comment:
This log might need to be moved to after the `if-block` since technically we **are** blocking instead of proceeding.

Reply:
Good point, logging improved in [4f2784d](https://github.com/opengovsg/FormSG/pull/6696/commits/4f2784d53cc3ad46cb1e54a85fe31f292c3a8fae)!